### ISSUE TITLE
Create Function to Calculate Haversine Distance

### ISSUE THREAD

**Issue by ZenithClown** [Astrodevil/Programming-Basics#299]
**Is your feature request related to a problem? Please describe.**
To measure distance b/w two points on earth, one uses the haversine distance. Create a `python` program to calculate haversine distance. This program is inspired from [scikit-learn distance metric](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise.haversine_distances.html).

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
